src/one_dragon_qt/view/setting/setting_push_interface.py

```
@@ -265,6 +268,27 @@ def _send_test_all_message(self):
+            if self.ctx.controller is None:
+                self.ctx.init_controller()
+
+            if self.ctx.controller is None:
+                return None
+
+            if not self.ctx.controller.is_game_window_ready:
+                if not self.ctx.controller.init_game_win():
+                    return None
+
+            _, screen = self.ctx.controller.screenshot(independent=True)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)